bin/pycbc_multi_inspiral

```diff
@@ -86,89 +110,254 @@ psd.insert_psd_option_group_multi_ifo(parser)
 scheme.insert_processing_option_group(parser)
 fft.insert_fft_option_group(parser)
 
+#Adding these to see if it lets me use the gating options
+from pycbc.vetoes.sgchisq import SingleDetSGChisq
+pycbc.opt.insert_optimization_option_group(parser)
+pycbc.weave.insert_weave_option_group(parser)
+pycbc.inject.insert_injfilterrejector_option_group(parser)
+SingleDetSGChisq.insert_option_group(parser)
 opt = parser.parse_args()
 
+generate_histograms = False
+generate_snr_plots = False
+generate_chi2_plots = False
+# Polarisation doesn't matter for coherent matched filter so set to zero.
+pol = 0
+# Use the time in the middle of the segment to calculate the antenna patterns.
+t_gps = (opt.trig_end_time[0] + opt.trig_start_time[0])/2
```

Comment:
Why this time? Shouldn't we use the trigger time for this?

Reply:
For the onsource it makes sense to use the trigger time, but that will cause problems when this is run on the offsource. Perhaps we could somehow set it to have the trigger time when running the on the onsource and use the middle of the segment otherwise?

Reply:
I'm a little confused now. In `coh_PTF` the antenna patterns are calculated using the time of the GRB and those same antenna patters used for the offsource ... This does mean that the RA changes, but that's okay, you want to keep the antenna patterns constant to assess the background.

So for the onsource the GRB time should be used, otherwise you will lose sensitivity.

For background the GRB time should also be used otherwise you will be calculating the appropriate background.

Reply:
Ok, that makes sense. I'll edit the setup to get the trigger time given to pycbc_multi_inspiral and we'll use that.